samples/msal-browser-samples/OfficeAddin/package.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "office-addin-taskpane-js",
+    "version": "0.0.1",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/OfficeDev/Office-Addin-TaskPane-JS.git"
+    },
+    "license": "MIT",
+    "config": {
+        "app_to_debug": "word",
+        "app_type_to_debug": "desktop",
+        "dev_server_port": 3000
+    },
+    "scripts": {
+        "build": "webpack --mode production",
+        "build:dev": "webpack --mode development",
+        "dev-server": "webpack serve --mode development",
+        "lint": "office-addin-lint check",
+        "lint:fix": "office-addin-lint fix",
+        "prettier": "office-addin-lint prettier",
+        "start": "office-addin-debugging start manifest.xml",
+        "start:desktop": "office-addin-debugging start manifest.xml desktop",
+        "start:web": "office-addin-debugging start manifest.xml web",
+        "stop": "office-addin-debugging stop manifest.xml",
+        "validate": "office-addin-manifest validate manifest.xml",
+        "watch": "webpack --mode development --watch"
+    },
+    "dependencies": {
+        "@azure/msal-browser": "file:../../../lib/msal-browser",
```

Comment:
This can reference the msal-browser version so that it can be consumed both locally via workspaces or via published npm. I'll leave it up to you if you think that would be useful

Reply:
Yes.  I see that the npm workspace is up and running now.  I'll update this.

Reply:
done.